### PR TITLE
Minor spelling and grammar corrections in Wanderers files

### PR DESCRIPTION
**Pull request by Fzzr**

### CODE REVIEW

File: data/wanderer ships.txt [hunk 1]
```diff
@@ -656,7 +656,7 @@ ship "Winter Gale"
 	explode "small explosion" 40
 	explode "medium explosion" 30
 	"final explode" "final explosion medium"
-	description "The Winter Gale was designed by the Wanderers as a hardier and better-armed alternative to their existing light patrol ships. The Gale's large number of weapon hardpoints makes it flexible enough to reconfigure for facing a wide variety of enemies."
+	description "The Wanderers designed the Winter Gale as a hardier and better-armed alternative to their existing light patrol ships. The Gale's large number of weapon hardpoints makes it flexible enough to reconfigure for facing a wide variety of enemies."
```
Comment (Bladewood) [inline data/wanderer ships.txt]: I'm not sure this is necessary. I prefer the older version.

**Comment by Fzzr** [comment]
Just a stylistic choice I prefer. Reverted.

File: data/wanderers start.txt [hunk 2]
```diff
@@ -1549,7 +1549,7 @@ event "capture of ik'kara'ka"
 		fleet "Large Unfettered" 400
 	planet "Varu Mer'ek"
 		attributes "unfettered" "factory"
-		description `This Wanderer factory world was recently captured by the Unfettered Hai, who are now busy trying to repurpose the Wanderer factories and equipment that were left behind, to build more ships for their war fleets. The Wanderers did not have time to demolish any of their equipment or to carry it off-world before the planet was lost. As a result, they have given the Unfettered a significant technological advantage here.`
+		description `This Wanderer factory world was recently captured by the Unfettered Hai, who are now busy trying to repurpose the Wanderer factories and equipment that were left behind to build more ships for their war fleets. The Wanderers did not have time to demolish any of their equipment or to carry it off-world before the planet was lost. As a result, they have given the Unfettered a significant technological advantage here.`
```
Comment (endless-sky) [inline data/wanderers start.txt]: The comma was necessary here because "who are now busy..." is a parenthetical phrase modifying "the Unfettered Hai."

Comment (Fzzr) [reply data/wanderers start.txt]: Ahhh! Okay, got it.

**Comment by Fzzr** [comment]
Woops, closed the wrong PR.

**Pull request merged**